features-support/hooks/ui-automation.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var WebDriver = require('selenium-webdriver');
+
+module.exports = function beforeHooks() {
```

Comment:
It's possible to lazily require webdriver inside the before hook so we don't pull in that code for non-webdriver tests. Require cashes it's results so there wouldn't be a performance hit for subsequent webdriver tests as long as they were started in the same process.

P.s. lazy requiring in Node in general is a bad idea because it's possible to create circular dependencies.